src/main/app/case/answers/possibleAnswers.ts

```diff
@@ -19,6 +19,8 @@ const IGNORE_UNREACHABLE_FIELDS = [
   'payments',
 ];
 
+const applicant2SequenceStart = '/applicant2/you-need-to-review-your-application';
```

Comment:
shouldn't you use the const in the url file?

Reply:
yeah, good point!